tests/test_services/test_llm_service.py

```
@@ -170,3 +170,274 @@ def test_利用可能モデル(self):
+                    from services.llm_service import LLMService
+
+                    service = LLMService()
+                    llm = service.get_or_create_model("gemini-1.5-flash")
+
+                    assert llm is not None
+                    # キャッシュされていることを確認
+                    assert "gemini-1.5-flash" in service.models
+
+    def test_キャッシュ済みモデル取得(self):
+        """キャッシュ済みモデルの取得"""
+        with patch("services.llm_service.genai") as mock_genai:
```

Ruff (0.14.6)

Local variable `mock_genai` is assigned to but never used

Remove assignment to unused variable `mock_genai`

(F841)